_pages/rfc-pairing.md

```diff
@@ -0,0 +1,74 @@
+branch, and your feature branch as the compare branch. Then, click the **Create pull request** button.
+* This should bring you to a page where you can write a description. Describe your problem, and what your pull request is aiming to
+solve. Indicate to reviewers the files and sections of code that you would like review on. After you are done, click **Create pull
+request.**
+* You should now be at a pull request page. Copy the link and paste it into the channel, asking for review. Indicate the languages and
+frameworks you are using.
+* Reviewers should go into the page, and select the Files changed tab. In this tab, one can see the diffs of the commits.
+Hovering your mouse over the diff should display a plus sign. Clicking on this plus sign will allow the reviewer to make line notes
+* As reviews come in, respond to code review by making appropriate code changes. You are also able to reply to inline comment by
+commenting on the same line. Talk it out, and don't be afraid to ask questions either on the pull request page, or on
+the pairing page. Push up any commits you make in the process.
+* Once reviewers feel good about the code, they can send a :+1: or a +1 to indicate the commit looks ready to be merged
+into master. They can do this by going to the main pull request page, scrolling to the bottom, entering text into the
+comment box, and pressing **Comment**.
+* Once you feel good about your pull request, you can feel free to merge it into master, either through the **Merge pull request**
+button on the pull request page, or through some other git interface of your choice; like, merging into master and pushing
+the changes back up on the command line.
+
+In this process, we ask that you respect all We All JS rules during code review. Additionally, remember to be constructive
+with your review, and to not be combatative when receiving review. We want to foster a place where less experienced coders
+can feel free to review and comment on the work of their more experienced peers.
+
+When considering whether or not you should code review something, the only thing we ask is that you try your best to be
+constructive. You do not need to review an entire PR, but we ask that, until code review is complete, you try as best as
+posssible to follow through on responding to any review you may have posted. 
+
+When considering posting a set of commits up for review, we ask that you try and remain focused in your commits. It is
+hard to review a massive set of changes, and review is most effective when reviewers are able to comprehensively review
+your code. When responding to code review, fear feature creep. Try your very best to not add any new features until code
+review is complete.
+
+## Live Pairing
+Live pairing involves two or more people on a call, looking at the same code. Feel free to put out calls for live pairing
```

Comment:
The use of "call" in two different contexts here can be confusing, is there another way to say "put out calls" or "put out a call" without using the word call? Or maybe the first usage could be changed to something else.